packages/core/README.md

```diff
@@ -83,7 +83,7 @@ Read [📽 the slides](http://slides.com/davidkhourshid/finite-state-machines) (
 - [The World of Statecharts](https://statecharts.github.io/) by Erik Mogensen
 - [Pure UI](https://rauchg.com/2015/pure-ui) by Guillermo Rauch
 - [Pure UI Control](https://medium.com/@asolove/pure-ui-control-ac8d1be97a8d) by Adam Solove
-- [Spectrum - Statecharts Community](https://spectrum.chat/statecharts) (For XState specific questions, please use the [GitHub Discussions](https://github.com/statelyai/xstate/discussions))
```

Comment:
Should the GitHub Discussions be removed from here?

Reply:
I'm not sure. I thought it would be easier to direct traffic to one place, our Discord. But we can have both or only the GitHub Discussions. What do you think?

Reply:
I'm in favor of keeping both. Let's wait on @davidkpiano to pitch in.

Reply:
Added a suggestion to keep the GitHub discussions link

Reply:
Thanks, I accepted the suggestion to keep both links 😊